Python/Magic Pen/magicpen.py

```diff
@@ -0,0 +1,74 @@
+#Importing Necessary projects
+import cv2
+import numpy as np
+#Creating the video capture object 
+cap = cv2.VideoCapture(0)
+
+#Defining upper and lower ranges for yellow color
+Lower = np.array([20, 100, 100])
+Upper = np.array([30, 255, 255])
+
+#Defining kernel for Morphological operators
+kernel = np.ones((5,5),np.uint8)
+#Defining starting point
+x0,y0 = -1,-1
+
+
+#Creating an empty image / white background with the same frame size
+temp = np.ones((480, 640, 3),dtype = np.uint8)
+temp = temp*255
+
+while True:
+        ret,frame = cap.read()
```

Comment:
Please run the Code through PyLint and change the variable names to something meaningful.